facade/baseFacade_test.go

```diff
@@ -238,6 +272,47 @@ func TestNewElrondProxyFacade_ShouldWork(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func TestNewElrondProxyFacade_GetBlocksByRound(t *testing.T) {
+	t.Parallel()
+
+	expectedResponse := &data.BlocksApiResponse{
+		Data: data.BlocksApiResponsePayload{
+			Blocks: []*data.Block{
+				{
+					Round: 1,
+				},
+				{
+					Round: 2,
```

Comment:
> Can you use the same round in the response and have for example the shardID different?

Yes, I've changed the `Hash` instead, to better reflect that blocks are different.

> Otherwise it might be confusing what the API does.

Good point, I really didn't realize how weird it looks :dagger: 

> It should return the blocks with the specified round, and here you return blocks with different rounds.

I've changed the tests to reflect the purpose better